commands/project/profile/add.ts

```diff
@@ -0,0 +1,268 @@
+import path from 'path';
+import fs from 'fs';
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import { getAccountId, getConfigAccounts } from '@hubspot/local-dev-lib/config';
+import { HsProfileFile } from '@hubspot/project-parsing-lib/src/lib/types';
+import { getAccountIdentifier } from '@hubspot/local-dev-lib/config/getAccountIdentifier';
+import {
+  getAllHsProfiles,
+  getHsProfileFilename,
+  loadHsProfileFile,
+} from '@hubspot/project-parsing-lib';
+import { trackCommandUsage } from '../../../lib/usageTracking';
+import { getProjectConfig } from '../../../lib/projects/config';
+import { uiBetaTag, uiAccountDescription } from '../../../lib/ui';
+import { uiLogger } from '../../../lib/ui/logger';
+import { EXIT_CODES } from '../../../lib/enums/exitCodes';
+import { YargsCommandModule, CommonArgs } from '../../../types/Yargs';
+import { makeYargsBuilder } from '../../../lib/yargsUtils';
+import { commands } from '../../../lang/en';
+import {
+  promptUser,
+  listPrompt,
+  confirmPrompt,
+} from '../../../lib/prompts/promptUtils';
+import { fileExists } from '../../../lib/validation';
+
+const command = 'add [name]';
+const describe = uiBetaTag(commands.project.profile.add.describe, false);
+const verboseDescribe = uiBetaTag(
+  commands.project.profile.add.verboseDescribe,
+  false
+);
+
+type ProjectProfileAddArgs = CommonArgs & {
+  name?: string;
+  targetAccount?: number;
+};
+
+async function handler(
```

Comment:
Could we split some of the `handler` code into smaller functions for readability?

Reply:
I moved all the logic to select a profile to copy the variables from into a dedicated util in the add.ts file. I think it should make the handler a littler easier to follow [7184fbf](https://github.com/HubSpot/hubspot-cli/pull/1520/commits/7184fbf223790b4f4f1246338071a5af248e9523)